src/app/modules/time-clock/components/entry-fields/entry-fields.component.ts

```diff
@@ -145,10 +157,20 @@ export class EntryFieldsComponent implements OnInit, OnDestroy {
       return;
     }
     this.entryForm.patchValue({ start_date: newHourEntered });
-    this.store.dispatch(new entryActions.UpdateCurrentOrLastEntry({ ...this.newData, ...this.entryForm.value }));
+    if (this.isTestUser) {
+      this.newData.owner_id = this.getOwnerId();
```

Comment:
Mmm I am thinking if you really need the owner_id. It should be already present in the data.

Reply:
You may realize that when you update an entry, you dont need to set the owner_id.

Reply:
Hi Roberto, thanks for your observations, the reason for sending the owner_id is because in the backend this property is used to query database. And when I tried connecting locally the backend didn't return anything because the property was not sent. Now with updateWntryRunning method I will see if I don't need to send this property. Thanks Roberto.